drake/systems/plants/collision/ResultCollector.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+#include "drake/systems/plants/collision/DrakeCollision.h"
+#include "ResultCollector.h"
+
+using namespace std;
+using namespace Eigen;
```

Comment:
"You may not use a using-directive to make all names from a namespace available."

https://google.github.io/styleguide/cppguide.html#Namespaces